### PR TITLE
CON-1922 - Security Foundations app documentation

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request (PR) renames the "[Security Analytics app](https://help.sumologic.com/docs/integrations/sumo-apps/security-analytics/)" to the "Security Foundations app" and documents the new dashboards:
https://help.sumologic.com/docs/integrations/sumo-apps/security-foundations/

Issue number: [CONNECT-1922](https://sumologic.atlassian.net/browse/CONN-1922)

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Comment by jpipkin1** [comment]
This PR is in draft until we get input from the SMEs.

**Comment by jpipkin1** [comment]
This PR is now ready for review.

**Review by kimsauce — APPROVED** [review]

**Comment by jpipkin1** [comment]
Thanks for the review, @kimsauce!

File: docs/integrations/sumo-apps/security-foundations.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+   * Proofpoint TAP
+* Network
+   * All firewall vendors
+   * All IDS/IPS vendors
+   * Cisco Umbrella
+
+
+## Install the Sumo Logic Foundations​ app
+
+The Security Foundations app is available to Sumo Logic Enterprise Security and Enterprise Suite accounts only, and requires a Professional Services contract to install and configure. For more information about this app and to request installation, send an email to `sales@sumologic.com`, contact your account manager, or see [Requesting Sumo Logic Enterprise Apps](/docs/integrations/sumo-apps/#Requesting-Sumo-Logic-Enterprise-Apps).
```
Comment (kimsauce) [inline docs/integrations/sumo-apps/security-foundations.md:37]: @jpipkin1 @oaklandersumo , typically we hyperlink email addresses; was it left unlinked intentionally?

Comment (jpipkin1) [reply docs/integrations/sumo-apps/security-foundations.md]: My bad, @kimsauce. It's fixed now.

**Comment by jpipkin1** [comment]
I put the PR back into draft mode. I may need to add more content.

**Comment by jpipkin1** [comment]
I moved the release note into this PR from PR #2623.

File: blog-service/2023-09-08-apps.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  - app catalog
+  - saas cloud
+  - security analytics
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We’re excited to announce the Sumo Logic Security Foundations app. This new app provides content to set up and execute threat detection alerts based on the MITRE framework, providing feature-packed dashboards that not only show you the most important alerts to investigate, but also the entities that pose the greatest risk. Here are some of the key features the new app offers:
```
Comment (kimsauce) [inline blog-service/2023-09-08-apps.md:15]:
```suggestion
We’re excited to announce the release of the Sumo Logic Security Foundations app. This new app provides content to set up and execute threat detection alerts based on the MITRE framework, providing feature-packed dashboards that not only show you the most important alerts to investigate, but also the entities that pose the greatest risk. Here are some of the key features the new app offers:
```

**Review by kimsauce — APPROVED** [review]
Looks good! Just had one small suggestion.

**Pull request merged**